node/src/GlideClient.ts

```diff
@@ -208,6 +207,10 @@ export class GlideClient extends BaseClient {
      *
      * @see {@link https://github.com/valkey-io/valkey-glide/wiki/General-Concepts#custom-command|Valkey Glide Wiki} for details on the restrictions and limitations of the custom command API.
      *
+     * @param args - A list of `function` arguments and it should not represent names of keys.
```

Comment:
this is not related to 'functions'.  
I would say: 
```
A list including the command name and arguments for the custom command.

Reply:
Updated